src/QMCDrivers/RMC/RMCFactory.cpp

```diff
@@ -30,6 +30,10 @@ namespace qmcplusplus
     int np = omp_get_max_threads ();
     //(SPACEWARP_MODE,MULTIPE_MODE,UPDATE_MODE)
     QMCDriver *qmc = 0;
+#ifdef QMC_CUDA
+    APP_ABORT("RMCFactory::create. RMC is not supported on GPU.\n");
```

Comment:
APP_ABORT will add a bit prefix and print as
Fatal Error. Aborting at RMCFactory::create. RMC is not supported on GPU.